### DISCUSSION TITLE
[项目组队]用于Web3Url的binding生成器

### DISCUSSION THREAD

**Discussion by nishuzumi** [IntensiveCoLearning/Web3-URL#149] [Ideas and Project Teams]
主要用于对Web3Url的模版生成，比如。
- 可以从开源合约中直接生成调用URL
- 可以直接生成模版代码，从调用URL中设定返回值，并且设定Typescript的类型
- 可以对一个合约的abi直接生成全套的模版调用代码

有感兴趣的小伙伴可以微信直接联系我： box1234567890

**Comment by luffythink** [comment]
⛽️
